### PR TITLE
Liftover

### PR DESCRIPTION
**Pull request by ch-kr**
Adds liftover functionality exclusively for gnomAD v2.1.1 MPC scores

### CODE REVIEW

File: rmc/resources/rmc.py [hunk 1]
```diff
@@ -742,6 +742,14 @@ def gnomad_fitted_score_path(
 This Table contains only one row per each unique locus/alleles combination.
 """
 
+mpc_liftover_release = TableResource(path=f"{MPC_PREFIX}/2.1.1/{CURRENT_FREEZE}/mpc.ht")
+"""
+Table containing gnomAD v2.1.1 MPC values lifted over to GRCh38.
+
+This is not a versioned resource;
+we created this resource specifically to liftover MPC to reference genome build 38.
```
[the remark this answers is not in the document]
Comment (ch-kr) [reply rmc/resources/rmc.py:750]: works for me

[the remark this answers is not in the document]
Comment (lw453) [reply rmc/resources/rmc.py:750]: Actually I apologize, I thought about this again and given that we may change RMC/MPC around in review, we shouldn't hardcode the freeze here and I think this specific line should be a versioned resource

[the remark this answers is not in the document]
Comment (ch-kr) [reply rmc/resources/rmc.py:750]: ooh, good point about the review. happy to revert -- will do so now!

File: rmc/utils/mpc.py [hunk 2]
```diff
@@ -1030,3 +1033,34 @@ def annotate_mpc(
...
+        " either direction)!"
+    )
+    if gnomad_version != "2.1.1":
+        raise DataException(
+            "This function was written to only liftover gnomAD v2.1.1 data from GRCh37"
+            " to GRCh38."
+        )
+    ht = hl.read_table(mpc_release.versions[freeze].path)
+    ht = default_lift_data(ht, remove_failed_sites=remove_failed_sites)
+    ht.write(mpc_liftover_release.path, overwrite=True)
```
Comment (lw453) [inline rmc/utils/mpc.py:1066]:
```suggestion
    ht.write(mpc_liftover_release.versions[freeze]path, overwrite=True)
```

**Review by lw453 — CHANGES_REQUESTED** [review]

**Review by lw453 — APPROVED** [review]

**Pull request merged**